apps/lib/s_socket.c

```diff
@@ -84,8 +84,10 @@ int init_client(int *sock, const char *host, const char *port,
     int ret;
     int options = 0;
 
-    if (tfo && ba_ret != NULL)
+    if (*ba_ret != NULL) {
```

Comment:
This check isn't required, please remove.

Reply:
Done